src/Sylius/Bundle/ResourceBundle/Grid/Renderer/TwigGridRenderer.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Grid\Renderer;
+
+use Sylius\Bundle\ResourceBundle\Grid\Parser\OptionsParserInterface;
+use Sylius\Component\Grid\Definition\Action;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\Definition\Filter;
+use Sylius\Component\Grid\Renderer\GridRendererInterface;
+use Sylius\Component\Grid\View\GridViewInterface;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+final class TwigGridRenderer implements GridRendererInterface
+{
+    /**
+     * @var GridRendererInterface
+     */
+    private $gridRenderer;
+
+    /**
+     * @var \Twig_Environment
+     */
+    private $twig;
+
+    /**
+     * @var OptionsParserInterface
+     */
+    private $optionsParser;
+
+    /**
+     * @var array
+     */
+    private $actionTemplates;
+
+    /**
+     * @param GridRendererInterface $gridRenderer
+     * @param \Twig_Environment $twig
+     * @param OptionsParserInterface $optionsParser
+     * @param array $actionTemplates
+     */
+    public function __construct(
+        GridRendererInterface $gridRenderer,
+        \Twig_Environment $twig,
+        OptionsParserInterface $optionsParser,
+        array $actionTemplates = []
+    ) {
+        $this->gridRenderer = $gridRenderer;
+        $this->twig = $twig;
```

Comment:
Yes, it is here: https://github.com/Sylius/Sylius/pull/6793/files#diff-bb793d8d4c25918c7655a9d11a4a218eR96